cli/script/command-executor.ts

```diff
@@ -224,6 +259,54 @@ function appRename(command: cli.IAppRenameCommand): Promise<void> {
         });
 }
 
+function addCollaborator(command: cli.ICollaboratorAddCommand): Promise<void> {
+    throwForInvalidEmail(command.email);
+
+    return getAppId(command.appName)
+        .then((appId: string): Promise<void> => {
+            throwForInvalidAppId(appId, command.appName);
+
+            return sdk.addCollaborator(appId, command.email)
+                .then((): void => {
+                    log("Successfully added the \"" + command.email + "\" app as a collaborator to \"" + command.appName +"\".");
```

Comment:
Shouldn't this be something like `Successfully added [EMAIL] as a collaborator to the [APP] app"?